### PR TITLE
Add Landing page, Login page, Sign up page 1 & 2, and basic navbar structure 

### PR DESCRIPTION
**Pull request by saraholiviawu**
**Landing page:** 

Description: The landing page will be the first page that a user sees when they search the website. The page includes a login button and an input box to submit an address.

Future steps: Current state is sufficient for MVP. Given time, I will add a dropdown when user clicks in the search box suggesting their current address / saved address. I will also add a map marker icon in the search box and text / images below the first page. 

**Authentication pages:** 

Description: User can navigate to and between these pages by clicking the Landing page navbar "Log in" option or by clicking links in Login Page / Sign up page 1 & 2 titled "Sign up" or "Sign in"

Future steps: Add authentication methods using Firebase API.

**Navbar (applied across multiple pages):** 

Description: The navbar presents different choices depending on the status (login and type of user) and user journey location of the user.

Landing page (index.html):  Ripple and Log in links.
Stories page (stories.html): Ripple, Today, Log in links
Stories page for Community Members (storiesCM.html): Ripple, Today, Avatar with drop down links to 'Your favorites,' 'Account settings,' 'Log out'
video snippet: https://drive.google.com/file/d/19fA4oSENPnjW1xiQZivNwJ-P61HhCjZe/view

Future steps: Add a search bar for logged in versions.

### CODE REVIEW

File: ripple/src/main/webapp/script.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// Display an alert containing the inputted address if user presses enter
+function searchAddress(e){
+	addressInput = document.getElementById("address-input").value;
```
Comment (sabadas) [inline ripple/src/main/webapp/script.js:3]: maybe a Javascript thing I am not familiar with, but shouldn't this and the line after the if block be indented to match the if block?

File: ripple/src/main/webapp/style.css [hunk 2]
```diff
@@ -0,0 +1,224 @@
+
+:root {
```
Comment (sabadas) [inline ripple/src/main/webapp/style.css:2]: great job on this! I think this makes it easier to make larger scale changes!

File: ripple/src/main/webapp/style.css [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+  /* Color variables */
+  --default-hover-seagreen-color: #87C4AD;
```
Comment (sabadas) [inline ripple/src/main/webapp/style.css:4]: Not really specific to this PR (and should not block it), but I noticed that a lot of these have similar values like `#707070`. Does it make sense to have like a primaryColor (or secondaryColor) defined? That might be a good question to ask UX and how they approach this. You want to make sure everyone is using the same vocab/names.

Comment (saraholiviawu) [reply ripple/src/main/webapp/style.css:4]: Waiting for response from UX. Will otherwise use primary / secondary naming practice.

**Pull request merged**